src/interactive.js

```diff
@@ -0,0 +1,40 @@
+import Methods from './utils.js';
+
+export default class Interactive {
+static toggleCompleted = (id, curStatus) => {
+  const todoList = Methods.getLocalStorageData();
+  todoList[id].completed = curStatus;
+  Methods.setLocalStorageData(todoList);
+  Methods.showTodoItems();
+}
+
+static addCheckEvent = () => {
+  document.querySelectorAll('.checkbox').forEach((box) => box.addEventListener('click', () => {
```

Comment:
- [Optional] Dear @V-Blaze, I see you use the `click` event. This works but a better event listener for this would be `change`, This will prevent the event from being triggered unless the value change (regardless of user click). Please consider replacing it.

Reply:
Thanks, the event type has been changed..